src/main/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserController.kt

```diff
@@ -323,6 +323,79 @@ class UserController(
     val groups = userGroupService.getMyAssignableGroups()
     return groups.map { UserGroupDto(it) }
   }
+
+  @GetMapping("/maintain/{userId}")
```

Comment:
Not sure on this url @karenmillermoj - would /maintain/email/{userId} be better?